unittest.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
Can you move this file into scripts/? Usually that's where things like this go, unless it's specific to an application or platform.